apps/ledger-live-mobile/src/components/Nft/NftGallery/ScrollToTopButton.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FC, useContext } from "react";
+import { Animated, FlatList, StyleSheet } from "react-native";
+import { WalletTabNavigatorScrollContext } from "../../WalletTab/WalletTabNavigatorScrollManager";
+import { useTranslation } from "react-i18next";
+import { Button } from "@ledgerhq/native-ui";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
+
+const ScrollToTopButton: FC = () => {
```

Comment:
Unless I'm missing something it seems like you are reinventing the wheel here, we have `scrollToTop` util which should take care of all the shenanigans of scrolling, I think you can just use that on your button instead of rewriting it.

Reply:
I had a look at this and it seems that it works if your view is wrapped in `NavigationScrollView` but the wallet/gallery isn't, so not sure i can use that util.

My understanding of how the wallet view works so far is that the NFT and Crypto tab share scroll behaviour in an attempt to keep them in sync (I assume for the tab and header animation) all driven by this context. I think I have to scroll this view manually using the ref.

Reply:
I'd ask for someone from live-hub to review this then 🙏🏼 but you have my ✔️

Reply:
I'm not expert with scrolls behaviour.
But yes Crypto Tab and NFTs tab are in sync when scrolling down.

So for the button i'm ok, i don't know if we have done already something similar. 

My concern is just about the sync  when scrolling up? what happen to the crypto tab? Is it in sync? :) If yes so LGTM !

Reply:
I've confirmed that the header animation is kept in sync when using this scroll to top button and navigating back tot he "Crypto" tab.